protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Unreleased
+
+* Update READMEs of `protobuf` and `protoc_plugin`:
+  * Use `dart pub` instead of `pub` in command examples (a7e75cb)
```

Comment:
Don't we have a PR to point to here instead of the commit?

Reply:
No, that commit was made to `master` directly.